apps/passport-client/src/dispatch.ts

```diff
@@ -756,11 +774,25 @@ async function doSync(
     state.pcds,
     state.subscriptions
   );
+
+  if (state.userInvalid) {
```

Comment:
Interesting.  I would've put this at the top of doSync so as to skip the whole process.  Is there a reason to keep download and feeds happening?  We don't really support the notion that the user ever becomes valid again without logging out.

Reply:
I'll put it at the top. I think I misinterpreted the above resolved comment - you meant for the early return to occur at the top, whereas I interpreted that you want it to occur at the place you left the comment.

Reply:
Right.  GitHub doesn't let me comment on unchanged lines.